src/app/query/bitcoin/bitcoin-client.ts

```diff
@@ -51,7 +51,45 @@ export interface BestinslotInscriptionsByTxIdResponse {
   blockHeight: number;
 }
 
-class BestinslotInscriptionsApi {
+interface Brc20TokenResponse {
+  ticker: string;
+  overall_balance: string;
+  available_balance: string;
+  transferrable_balance: string;
+  image_url: string | null;
+}
+
+export interface Brc20Token extends Brc20TokenResponse {
+  decimals: number;
+  holderAddress: string;
```

Comment:
Do we typically mix response types with camelCased props in client types?

Reply:
don't understand what you mean here :( 
I just need to extend response token type here to use in the app